bin/load-mocks

```diff
@@ -759,6 +718,131 @@ def main(num_events=1, extra_events=False, load_transactions=False):
+                measurements={
+                    "fp": {"value": random_normal(1250 - 50 * day, 200, 500)},
+                    "fcp": {"value": random_normal(1250 - 50 * day, 200, 500)},
+                    "lcp": {"value": random_normal(2800 - 50 * day, 400, 2000)},
+                    "fid": {"value": random_normal(5 - 0.125 * day, 2, 1)},
+                },
+                span_id=frontend_transaction_span_id,
+                # Root
+                parent_span_id=None,
+                trace=trace_id,
+                spans=[
+                    {
+                        "same_process_as_parent": True,
+                        "op": "http",
+                        "description": "GET /api/plants/?all_plants=1",
+                        "parent_span_id": frontend_transaction_span_id,
+                        "data": {
+                            "duration": random_normal(
+                                1 - 0.05 * day, 0.25, 0.01, frontend_duration / 1000
+                            ),
+                            "offset": 0.02,
+                        },
+                        "span_id": frontend_span_id,
+                        "trace_id": trace_id,
+                    }
+                ],
+            )
+            # try to give clickhouse some breathing room
+            if slow:
+                time.sleep(0.05)
+
+            backend_span_ids = [
+                (name, uuid4().hex[:16])
+                for name in ["/products/all/", "/analytics/", "tasks.create_invoice"]
+            ]
+            backend_duration = random_normal(1500 + 50 * day, 250, 500)
+
+            create_sample_event(
+                project=backend_project,
+                platform="transaction",
+                transaction="/api/plants/",
+                event_id=uuid4().hex,
+                user=transaction_user,
+                timestamp=timestamp,
+                start_timestamp=timestamp - timedelta(milliseconds=backend_duration),
+                # match the trace from the javascript transaction
+                trace=trace_id,
+                parent_span_id=frontend_span_id,
+                spans=[
+                    {
+                        "same_process_as_parent": True,
+                        "op": "http",
+                        "description": name,
+                        "parent_span_id": frontend_span_id,
+                        "data": {
+                            "duration": random_normal(
+                                0.75 - 0.05 * day, 0.25, 0.01, backend_duration / 1000
+                            ),
+                            "offset": 0.02,
+                        },
+                        "span_id": backend_span_id,
+                        "trace_id": trace_id,
+                    }
+                    for name, backend_span_id in backend_span_ids
+                ],
+            )
+            if slow:
```

Comment:
When would you run this without slow?

Reply:
I... will add some docs explaining this, but basically depends on your personal computer. I found that I could run `--load-transactions` consistently without crashing, but @Zylphrex has reported that running it crashes clickhouse consistently 😞